lib/mongoose_push.ex

```diff
@@ -63,6 +63,7 @@ defmodule MongoosePush do
         request
         |> Map.put(:alert, request[:alert])
         |> Map.put(:data, request[:data])
+        |> Map.put(:time_to_live, request[:time_to_live])
```

Comment:
Why do we need it here? `alert` and `data` need this to ensure those keys are in `request` map, but I don't see any logic in this PR that would require `time_to_live` to be present regardless of whether it's set or not.